src/components/ModalVote.vue

```diff
@@ -245,12 +245,14 @@ watch(
         <template
           v-else-if="isValidationAndPowerLoaded && !isValidationAndPowerLoading"
         >
-          <BaseMessageBlock v-if="hasVotingPowerFailed" level="warning">
-            {{ t('votingPowerFailedMessage') }}
-          </BaseMessageBlock>
-          <BaseMessageBlock v-if="hasVotingValidationFailed" level="warning">
-            {{ t('votingValidationFailedMessage') }}
-          </BaseMessageBlock>
+          <template v-if="hasVotingPowerFailed || hasVotingValidationFailed">
+            <BaseMessageBlock v-if="hasVotingPowerFailed" level="warning">
+              {{ t('votingPowerFailedMessage') }}
+            </BaseMessageBlock>
+            <BaseMessageBlock v-if="hasVotingValidationFailed" level="warning">
+              {{ t('votingValidationFailedMessage') }}
+            </BaseMessageBlock>
+          </template>
```

Comment:
```suggestion

            <BaseMessageBlock v-if="hasVotingPowerFailed" level="warning">
              {{ t('votingPowerFailedMessage') }}
            </BaseMessageBlock>
            <BaseMessageBlock v-else-if="hasVotingValidationFailed" level="warning">
              {{ t('votingValidationFailedMessage') }}
            </BaseMessageBlock>

```

Reply:
@ChaituVR would prefer using `else-if` instead of using another block

Reply:
I don't think there is the need to show vp and validation error at the same time